packages/form-js-viewer/test/spec/import/Importer.spec.js

```diff
@@ -309,6 +309,32 @@ describe('Importer', function() {
       });
     }));
 
+    it('should sanitize data', inject(async function(form) {
```

Comment:
I'd love to have dedicated test cases for various aspects here (i.e. clearly showcase static vs. dynamic fields).

Let's break up this test case.

Reply:
Fair point 👍